service/src/main/scala/org/apache/celeborn/server/common/container/DefaultContainerInfoProvider.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.server.common.container
+
+import org.apache.celeborn.common.util.Utils
+import org.apache.celeborn.rest.v1.model.ContainerInfo
+
+class DefaultContainerInfoProvider extends ContainerInfoProvider {
+
+  override def getContainerInfo(): ContainerInfo = {
```

Comment:
@akpatnam25 I can see that there are no dependency issues. Is there any particular reason to get an instance of `DefaultContainerInfoProvider`? If there is no relative issues, I think this reflect invoke can be replaced by direct invoke.

Reply:
responded below for this